src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupName;
+
+class AddCustomerGroupCommand
+{
+    /** @var string[] */
+    private $localizedNames;
+    /** @var float */
+    private $reduction;
+    /** @var int */
+    private $priceDisplayMethod;
+    /** @var bool */
+    private $showPrice;
+
+    /**
+     * @param string[] $localizedNames
+     * @param float $reduction
+     * @param int $priceDisplayMethod
+     * @param bool $showPrices
+     */
+    public function __construct(
+        array $localizedNames,
+        float $reduction,
+        int $priceDisplayMethod,
+        bool $showPrices
```

Comment:
showPrice or showPrices? I suggest leaving it same for construct  argument and class property

Reply:
showPrice, it was a typo nice catch fixed